guessing_game_v2.html

```diff
@@ -31,92 +30,68 @@
     }
     oneOrZero();
```

Comment:
I am not sure why that is show that way in this version of my code? But if you check the latest version of It below you'll see that it creates a random three digit array of zero and one.
https://github.com/brain11f/CodeFellowsHomeWork_day1/blob/master/guessing_game_v2.html